### PR TITLE
[W5.10h][W13-A1]Raynold Ng 

### PR DESCRIPTION
**Pull request by raynoldng**
LO-UseCases

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -88,6 +88,16 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use Case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of tags
+3. User requests to rename a specific tag in the list
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:97]: Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -88,6 +88,16 @@ Use case ends.
...
+4. User enters new name for the tag
+5. AddressBook confirms the change with the user before renaming the tag
+
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:100]: - What happens when the new tag is already present in the address book?
- What happens if the new tag is same as the old tag?

**Comment by okkhoy** [comment]
@raynoldng Please think about extensions as well when you write use cases.

Some comments added. Please ack & close the PR after reading comments.

**Comment by raynoldng** [comment]
Acknowledged with thanks

**Pull request closed without merging**